2005-05-05  Nick Clifton  <nickc@redhat.com>

	* config/tc-arm.c (arm_opts): Make -mlittle-endian switch set
	the target_big_endian variable to false.

===================================================================
RCS file: /cvs/src/src/gas/config/tc-arm.c,v
retrieving revision 1.202
retrieving revision 1.203
diff -u -r1.202 -r1.203
--- src/gas/config/tc-arm.c	2005/04/19 15:05:08	1.202
+++ src/gas/config/tc-arm.c	2005/05/05 07:40:56	1.203
@@ -12870,7 +12870,7 @@
   {"mapcs-reentrant", N_("re-entrant code"), &pic_code, 1, NULL},
   {"matpcs", N_("code is ATPCS conformant"), &atpcs, 1, NULL},
   {"mbig-endian", N_("assemble for big-endian"), &target_big_endian, 1, NULL},
-  {"mlittle-endian", N_("assemble for little-endian"), &target_big_endian, 1,
+  {"mlittle-endian", N_("assemble for little-endian"), &target_big_endian, 0,
    NULL},
 
   /* These are recognized by the assembler, but have no affect on code.  */
